scss/_patterns_subnav.scss

```diff
@@ -0,0 +1,57 @@
+@import 'settings';
+
+@mixin vf-p-subnav {
+  .p-subnav {
+    position: relative;
+
+    > a {
```

Comment:
You can use `p-subnav__item` now

Reply:
I wish I could but the defaul navigation pattern is greedy. I'll look and see if there's a quick fix to the navigation pattern that could fix this.

Reply:
Issue created: https://github.com/vanilla-framework/vanilla-framework/issues/2168